events/models.py

```diff
@@ -1,29 +1,33 @@
-from django.db import models
 from django.contrib.auth import get_user_model
+from django.db import models
 from django.utils.translation import gettext_lazy as _
 
 User = get_user_model()
 
+
 class Tag(models.Model):
+    """ Model to store tags for events """
```

Comment:
Should we ignore the enforcement to write doc strings? While it is a good idea in theory, My opinion is that doc strings and comments should only be placed where complex business logic is present, Code and naming conventions should be reviewed strictly and be good enough that they explain their purpose.

Reply:
I agree, but I would argue that we should use docstrings to document our code. I know the documentation we have right now seems redundant, but we will make efforts to improve it, and then we can use tools like `Sphinx` to generate robust documentation.